spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -306,7 +306,7 @@ static MimeType mimeTypeFromObject(Object payload, String originalContentType) {
 				return MimeTypeUtils.APPLICATION_OCTET_STREAM;
 			}
 			if (payload instanceof String) {
-				return MimeTypeUtils.APPLICATION_JSON_VALUE.equals(originalContentType) ? MimeTypeUtils.APPLICATION_JSON
+				return (originalContentType.startsWith(MimeTypeUtils.APPLICATION_JSON_VALUE)) ? MimeTypeUtils.APPLICATION_JSON
```

Comment:
OK. My `originalContentType` is `application/json;charset=UTF-8`, but according the outcome of this operation I'll get only `application/json`.
Since the `param` maybe any, wouldn't it be better to return `originalContentType` instead of raw `MimeTypeUtils.APPLICATION_JSON`?